test/snippet/stream/transparent_istream.cpp

```diff
@@ -0,0 +1,47 @@
+#include <fstream>
+
+#include <bio/stream/transparent_istream.hpp>
+#include <bio/stream/transparent_ostream.hpp>
+
+int main()
+{
+    { // create my_file.txt and my_cin_file.txt so the code below works
+        std::ofstream os{"my_file.txt"};
+        os << "Foo";
+        std::ofstream os2{"my_cin_file.txt"};
+        os2 << "Foo";
+    }
```

Comment:
Are these files deleted automatically?

Reply:
nope :D they appear in the `build/dir/of/snippet/tests` so I thought it's not so bad

Reply:
That's not great, but it is better than having weird filenames in the  snippet